File: Python/0892.py
seasons = {
    1: 'Winter',
    2: 'Winter',
    3: 'Spring',
    4: 'Spring',
    5: 'Spring',
    6: 'Summer',
    7: 'Summer',
    8: 'Summer',
    9: 'Autumn',
    10: 'Autumn',
    11: 'Autumn',
    12: 'Winter',
}

nMonth = int(input())

season = seasons.get(nMonth, None)

if season is not None:
    print(season)
else:
    print('Error')

File: C++/0892.cpp
#include <iostream>
using namespace std;


int main()
{
    int nMonth;

    cin >> nMonth;

    if (nMonth <= 0 || nMonth > 12)
        cout << "Error" << endl;
    else if (nMonth <= 2 || nMonth == 12)
        cout << "Winter" << endl;
    else if (nMonth <= 5)
        cout << "Spring" << endl;
    else if (nMonth <= 8)
        cout << "Summer" << endl;
    else
        cout << "Autumn" << endl;

    return 0;
}
